src/codeql.ts

```diff
@@ -212,6 +212,19 @@ export interface CodeQL {
   resolveExtractor(language: Language): Promise<string>;
 }
 
+export interface VersionOutput {
+  productName: string;
+  vendor: string;
+  version: string;
+  sha: string;
+  branches: string[];
+  copyright: string;
+  unpackedLocation: string;
+  configFileLocation: string;
+  configFileFound: boolean;
+  features?: { [name: string]: boolean };
```

Comment:
We can simplify this down to just the entries the CodeQL Action cares about:

```suggestion
  version: string;
  features?: { [name: string]: boolean };
```